docs/development/pigs/pig-005.rst

```diff
@@ -0,0 +1,183 @@
+.. include:: ../../references.txt
+
+.. _pig-005:
+
+***************************
+PIG 5 - Gammapy 1.0 Roadmap 
+***************************
+
+* Author: Axel Donath (editor), Régis Terrier & Christoph Deil
+* Created: September 28, 2018
+* Accepted: -
+* Status: open
+* Discussion: -
+
+Introduction
+============
+This PIG describes the required short- and medium-term **development work up to
+the Gammapy 1.0** release. The anticipated time scale for this development effort
+is **9 - 12 months** and will be concluded by the Gammapy 1.0 release in fall 2019.
+The question of **API design and sub-module structure for Gammapy 1.0 will be
+addressed in a separate PIG**. 
+
+The content of this document was decided based upon user feedback from the first
+CTA data challenge (DC1), experience from analysing existing datasets as well as
+definition of use cases (see below). The content will be **updated in the comming month**
+and be adjusted to upcoming **requirements defined by CTA**. The document will
+result in a series of subsequent PIGs, which are written and implemented by contributing
+developers, that **take responsibility for one of the projects** described below.
+
+
+Releases (add responsible person)
+=================================
+Up to the Gammapy 1.0 release we will pursue a **shorter release cycle**, with
+releases  **every 2-3 months**. The following releases are planned:
+
+* Gammapy 0.9 in November 2018
+* Gammapy 0.10 in January 2019
+* Gammapy 0.11 in March 2019
+* Gammapy 0.12 in Mai 2019
+* Gammapy 0.13 in Juli 2019
+* Gammapy 1.0 in October 2019
+
+The exact schedule is flexible, as it depends on when features are ready to ship.
+With this process we aim to enhance user feedback as well as set intermediate
+**milestones for the development progress**. We look for a **release manager**
+to take over the responsibility for the release process.
+
+
+Meetings
+========
+We plan to hold **three coding sprints** up to the Gammapy 1.0 release. If you would
+like to host one of the sprints get in contact with the Gammapy coordination
+committee (gammapy-coordination-l@in2p3.fr).
+
+We plan to continue the **weekly developers calls** every Friday 10 am. In addition
+we could start **monthly Gammapy user calls**, for regular user support and feedback.
+(to be discussed).
+
+We plan to hold **Gammapy workshops and tutorials** at upcoming science and
+collaboration meetings (to be disccused). 
+
+
+Use Cases
+=========
+The following use cases should be supported by Gammapy 1.0:
+
+* Fit 1D spectrum and 3D cube simultaneously
+* Further definitions defined by CTA (add link).
```

Comment:
@cboisson - Do you have a link to a CTA document containing a description of use cases for the CTA ST?
It's OK if it's a CTA-internal document, we can still link to it from here, most people are in CTA.

I don't think we should extend the use cases section in this PIG significantly, maybe add one or two bullets with use cases that aren't already possible, but trying to repeat the description of use cases here would be a huge time sink.

Reply:
Here is the one prepared some time ago:
https://forge.in2p3.fr/projects/observer-access-use-cases/wiki/Science_Tools_Use-Cases_